Mathlib/SetTheory/Cardinal/Ordinal.lean

```diff
@@ -1445,3 +1447,46 @@ theorem extend_function_of_lt {α β : Type _} {s : Set α} (f : s ↪ β) (hs :
 -- end Bit
 
 end Cardinal
+
+section Initial
+
+namespace Ordinal
+
+/--
+`ω_ o` is a notation for the *initial ordinal* of cardinality
+`aleph o`. Thus, for example `ω_ 0 = ω`.
+-/
+scoped notation "ω_" => λ o ↦ ord <| aleph o
```

Comment:
```suggestion
scoped notation "ω_" o => ord <| aleph o
```
this adds an extra beta redex in uses of the notation, plus using `ω_` without applying it to something looks weird and confusing

Reply:
For the first thing, that's simply me not knowing how to use the `notation` command, sorry for that.

For the notation itself, I do not like `ω_` either; it was a compromise solution [discussed on Zulip](https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/Defining.20initial.20ordinals/near/338425902). I would have liked to have a "subscript argument", but from that discussion I understood that it is not possible.

Do you suggest that the `ωₙ` notation be introduced on demand?

Reply:
Let's commit the suggestion and call it a day :) I think Mario is saying that a standalone `ω_` looks weird but `ω_ o` looks okay. I think it's definitely reasonable to introduce local notations `ω₁`, `ω₂` etc. to make things more readable, but if you want to use the `Ordinal.initial` function rather than just one of its values then you can't possibly fit an arbitrary expression (or even character) into the subscript, and would have to write `ω_ expr`.

Reply:
Of course I accept the suggestion, I honestly wanted to know if it was deemed even better not to introduce `ω_` at all.